content/en/launch/integration-guides/dagster.md

```diff
@@ -317,9 +317,9 @@ Supported properties:
 * `type`: (str) The type of the artifact, which is used to organize and differentiate artifacts. Common types include dataset or model, but you can use any string containing letters, numbers, underscores, hyphens, and dots. Required when the output is not already an Artifact.
 * `description`: (str) Free text that offers a description of the artifact. The description is markdown rendered in the UI, so this is a good place to place tables, links, etc.
 * `aliases`: (list[str]) An array containing one or more aliases you want to apply on the Artifact. The integration will also add the “latest” tag to that list whether it’s set or not. This is an effective way for you to manage versioning of models and datasets.
-* [`add_dirs`]({{< relref "/ref/python/artifact.md#add_dir" >}}): (list[dict[str, Any]]): An array containing configuration for each local directory to include in the Artifact. It supports the same arguments as the homonymous method in the SDK.
-* [`add_files`]({{< relref "/ref/python/artifact.md#add_file" >}}): (list[dict[str, Any]]): An array containing configuration for each local file to include in the Artifact. It supports the same arguments as the homonymous method in the SDK.
-* [`add_references`]({{< relref "/ref/python/artifact.md#add_reference" >}}): (list[dict[str, Any]]): An array containing configuration for each external reference to include in the Artifact. It supports the same arguments as the homonymous method in the SDK.
+* [`add_dirs`]({{< relref "/ref/python/sdk/classes/artifact#add_dir" >}}): (list[dict[str, Any]]): An array containing configuration for each local directory to include in the Artifact. It supports the same arguments as the homonymous method in the SDK.
```

Comment:
```suggestion
* [`add_dirs`]({{< relref "/ref/python/sdk/classes/artifact#add_dir" >}}): (list[dict[str, Any]]): An array containing configuration for each local directory to include in the Artifact. It supports the same arguments as the same named method in the SDK.
```
"homonymous" is a crazy high reading comprehension level word. More instances below.

Reply:
Oh gosh. This must've slipped the cracks when the eng created the PR way back when.